src/components/App/App.css

```diff
@@ -1,5 +1,5 @@
-@import "./NotFoundPage.css";
-@import "./Header.css";
+@import './NotFoundPage.css';
```

Comment:
Prettier seems to [use double quotes by default](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEABAlgWwA4QE4wAEA5AB4B0YAzlcQNwgA0IE2M60VyoAhnnhADuABT4IuKHgDcI6ACZMQPKjGQAzHgBsqcZgCM8PMAGs4MAMrYj6KAHNkMPAFddIORDDqtO5jZ0FhQ1tMHi9tVwArKlIAIUMTM3MeTDgAGRs4MJ8WJxhsXIAmLNcrPH9kcBpFbDwbGAB1eRgAC2QADgAGZhqIHXrDbAqauH8pTOY8OABHJ3RJwJ5g0KQNcOYdTHQHZ1cqG1tNOABFJwh4YuYYHj1GuRbkAsvDdE19gGEITBCKqGhxkCcOgAKtcJKsdABfCFAA) (at least within the playground). It feels odd that it has converted these quotes to single ones. Is there a configuration option not shown in this PR that could be causing this?

Reply:
Prettier's default for quotes is double, so I'm not sure why they choose to change that for imports: https://prettier.io/docs/en/options.html#quotes.

Reply:
I was wondering about that - I'd like to see us go with single quotes everywhere except JSX attributes.